pkgs/dkg/drand.go

```diff
@@ -477,6 +480,19 @@ func (o *LocalOwner) postReshare(res *kyber_dkg.OptionResult) error {
 		o.broadcastError(err)
 		return fmt.Errorf("partial owner + nonce signature isnt valid %x", sigOwnerNonce.Serialize())
 	}
+	encInitPub, err := crypto.EncodePublicKey(o.InitiatorPublicKey)
+	if err != nil {
+		o.broadcastError(err)
+		return err
+	}
+	dataToSign := make([]byte, len(secretKeyBLS.Serialize())+len(encInitPub))
+	copy(dataToSign[:len(secretKeyBLS.Serialize())], secretKeyBLS.Serialize())
+	copy(dataToSign[len(secretKeyBLS.Serialize()):], encInitPub)
+	ceremonySig, err := o.signFunc(dataToSign)
+	if err != nil {
+		o.broadcastError(err)
+		return err
+	}
```

Comment:
as @MatusKysel mentioned earlier, this part repeats itself so I'd move it to a function to make sure it runs the same everywhere